app/commands.py

```diff
@@ -106,7 +106,7 @@ def purge_functional_test_data(user_email_prefix):
     """
     Remove non-seeded functional test data
 
-    users, services, etc. Give an email prefix. Probably "notify-test-preview".
+    users, services, etc. Give an email prefix. Probably "notify-tests-preview".
```

Comment:
Do we need the word "Probably" here?

Reply:
yup. you _might_ want to run it on something else, for example if you were running from your local but pointed at preview

Reply:
but most of the time it's (infrequently) used is to clear out data cos theres too much and we weren't anticipating the weird data. in this case, preview has 2705 org users and that's quite high (prod has 17 😂 ).

Reply:
Ahh, cheers, I get it now ☺️